app/assets/javascripts/reducers/stories.js

```diff
@@ -28,6 +29,12 @@ const initialState = {
 const storiesReducer = (state = initialState, action) => {
   switch (action.type) {
     case actionTypes.RECEIVE_STORIES:
+      if (action.from === EPIC) {
```

Comment:
Instead of doing this, can we create a new function like these? https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/models/beta/column.js#L13